courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb

```diff
@@ -0,0 +1,937 @@
+{
```

Comment:
<p>We typically enumerate learning objectives, so that the TODO's can mirror student activities. We don't have a standard, but ASL (and TCD's) use enumeration.</p><p> </p><ol><li>Apply techniques to clean and inspect data.</li><li>Split dataset into training, validation and test datasets.</li><li>Use the tf.data.Dataset to create an input pipeline.</li><li>Define, compile and train a model using the Keras Sequential API.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>